### DISCUSSION TITLE
Indicate 2.0 as a requirement

### DISCUSSION THREAD

**Discussion by twopiearr** [civitai/civitai#83] [Ideas]
New models are starting to appear based on the 2.x models from Stability AI. Since these models require changes to the code base, a lot of implementations don't currently support them. It would be GREAT if we could get an indicator of some kind on models that require 2.0 compatibility.

**Comment by JustMaier** [comment]
Gonna flag this is a bug since not having this essential breaks things for people.

**Comment by JustMaier** [comment]
**This is live!**

Now when uploading a model you can specify the base model for the version:
![example of selecting base model](https://user-images.githubusercontent.com/607609/208995060-27a4be2f-093c-435a-9ee5-b5821e4a73a4.png)

And we've added the ability to filter based on that data:
![example of filtering](https://user-images.githubusercontent.com/607609/208995116-7303dd92-40d8-48a8-816c-6313e350d5f2.png)

Example of the base model being displayed:
![image](https://user-images.githubusercontent.com/607609/208995346-ff13dcbb-5f45-4554-b12c-8bbbd592f476.png)
